main.tf

```diff
@@ -12,11 +12,6 @@ locals {
     )
   )
 
-  topic_name = length(var.existing_pub_sub_topic_id) > 0 ? var.existing_pub_sub_topic_id : "${var.prefix}-lacework-topic-${random_id.uniq.hex}"
-  topic_id = "projects/${local.project_id}/topics/${local.topic_name}"
-  subscription_name = length(var.existing_pub_sub_subscription_name) > 0 ? var.existing_pub_sub_subscription_name : "${var.prefix}-${local.project_id}-lacework-subscription-${random_id.uniq.hex}"
-  subscription_id = "projects/${local.project_id}/subscriptions/${local.subscription_name}"
```

Comment:
Hi @Manan-Bhatia-0 I'm missing some context on this. Could you add to the PR description why these local variables being removed. Thanks!